### PR TITLE
Announcing Scala.js 0.6.3.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @gzm0 and @nicolasstucki 
Scheduled for announcement tomorrow morning CET, assuming everything goes well with the CI.

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+We are excited to announce the release of Scala.js 0.6.3!
+
+A lot has happened in this release, probably because we dragged it for a longer amount of time than usual (2 months since the last release!).
+Besides the usual bag of bug fixes, there are three highlights:
+
+* Support for `BigInteger` and `BigDecimal`, thanks to an incredible amount of work by [@InTheNow](https://github.com/InTheNow).
+* Extension of `jsDependencies` to specify minified versions of the JS libraries.
+* *Experimental* output modes to target ECMAScript 6, as well as [Strong Mode](https://developers.google.com/v8/experiments).
+
+We also welcome Nicolas Stucki aka [@nicolasstucki](https://github.com/nicolasstucki) in the core team at EPFL.
```
Comment (gzm0) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: to the core team?

Comment (sjrd) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: you mean replace "in" by "to"?

Comment (gzm0) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: Yes... but I'm not sure.

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+* [#1490](https://github.com/scala-js/scala-js/issues/1490) JVM style module initialization under semantic flag
+* [#1512](https://github.com/scala-js/scala-js/issues/1512) Atomic write of `-fastopt.js` and `-opt.js`
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
+* [#1560](https://github.com/scala-js/scala-js/issues/1560) `java.io.StringReader.read(c:Array[Char],offset:Int,len:Int)` returns 0 at eof, instead of -1
+* [#1564](https://github.com/scala-js/scala-js/issues/1564) Allow overriding path prefixes
+* [#1569](https://github.com/scala-js/scala-js/issues/1569) AssertionError when crossProject is used as a ProjectRef in another project
+* [#1578](https://github.com/scala-js/scala-js/issues/1578) Report the full stack trace for exceptions thrown in Futures
+* [#1582](https://github.com/scala-js/scala-js/issues/1582) Parsing hexadecimal Long that has 0x80000000 bit set fails
+* [#1589](https://github.com/scala-js/scala-js/issues/1589) Switch statement with "if" causes compiler to fail
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
```
Comment (gzm0) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: scala-js/scala-js #1625 also went in.

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md (hunk 1 above)
Comment (sjrd) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: We can use "into" ^^
Joke aside, Google seems to agree with you.

**Comment by gzm0** [comment]
That's all. LGTM otherwise.

**Comment by sjrd** [comment]
Updated.

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md (hunk 2 above)
Comment (sjrd) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: Added below this line.

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+* [#1490](https://github.com/scala-js/scala-js/issues/1490) JVM style module initialization under semantic flag
+* [#1512](https://github.com/scala-js/scala-js/issues/1512) Atomic write of `-fastopt.js` and `-opt.js`
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
+* [#1560](https://github.com/scala-js/scala-js/issues/1560) `java.io.StringReader.read(c:Array[Char],offset:Int,len:Int)` returns 0 at eof, instead of -1
+* [#1564](https://github.com/scala-js/scala-js/issues/1564) Allow overriding path prefixes
+* [#1569](https://github.com/scala-js/scala-js/issues/1569) AssertionError when crossProject is used as a ProjectRef in another project
+* [#1578](https://github.com/scala-js/scala-js/issues/1578) Report the full stack trace for exceptions thrown in Futures
+* [#1582](https://github.com/scala-js/scala-js/issues/1582) Parsing hexadecimal Long that has 0x80000000 bit set fails
+* [#1589](https://github.com/scala-js/scala-js/issues/1589) Switch statement with "if" causes compiler to fail
```
Comment (nicolasstucki) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: Use `if` instead of "if".

File: _posts/news/2015-05-12-announcing-scalajs-0.6.3.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+* [#1556](https://github.com/scala-js/scala-js/issues/1556) Test adapter provisions slaves too aggressively
+* [#1625](https://github.com/scala-js/scala-js/issues/1625) Emit warning if js.Dynamic.literal.applyDynamicNamed has duplicate keys (and [#1595](https://github.com/scala-js/scala-js/issues/1595) not emit illegal JS in that situation)
```
Comment (nicolasstucki) [inline _posts/news/2015-05-12-announcing-scalajs-0.6.3.md]: Use `js.Dynamic.literal.applyDynamicNamed` instead of js.Dynamic.literal.applyDynamicNamed.

**Comment by nicolasstucki** [comment]
Thats all. 
LGTM

**Comment by sjrd** [comment]
Updated.

**Pull request merged**